doc/cephfs/fs-volumes.rst

```diff
@@ -91,12 +91,8 @@ List subvolume groups using::
 
     $ ceph fs subvolumegroup ls <vol_name>
 
-Create a snapshot (see :doc:`/cephfs/experimental-features`) of a
-subvolume group using::
-
-    $ ceph fs subvolumegroup snapshot create <vol_name> <group_name> <snap_name>
-
-This implicitly snapshots all the subvolumes under the subvolume group.
+.. note:: Subvolume group snapshot feature is no longer supported in mainline CephFS (existing group
```

Comment:
Possibly need this one word change here,

```suggestion
.. note:: Subvolume group snapshot feature is no longer supported in nautilus CephFS (existing group
```

Reply:
Go ahead and push a follow-up commit with that change.